docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,182 @@
+ZAAKTYPEN API
+===============================================
+**FUNCTIONELE DOCUMENTATIE**
+
+| **Aspect**      | **Beschrijving**                                                                                                                                                                                  |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Doel            | Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van zaaktypen en de zaaktypecatalogi (ZTC) waarvan zij deel uit maken. |
+| Domein          | Zaakgericht werken                                                                                                                                                                                |
+| Provider        | [Zaaktypecataloguscomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-3ef9cdd9-631c-4d3e-88c3-f756423d6314) (GEMMA2)                                                              |
+| Consumer        | Componenten waarmee zaaktypen beheerd worden en componenten die zaaktypegegevens gebruiken voor het behandelen van zaken, zoals Zaakafhandelcomponenten (ZAC’s).                                                              |
+| Informatiemodel | [IMZTC, versie 2.2](https://www.gemmaonline.nl/images/gemmaonline/1/1d/EgIM_20180621_-_ag_4_-_GEMMA_ImZTC_2.2_-_CONCEPT_20180613.zip) (in-ontwikkeling)                                         |
+| Specificaties   | <https://ref.tst.vng.cloud/zrc/api/v1/schema>                                                                                                                                                     |
+| Bijzonderheden  | -                                                    |
+
+Deze API omvat (vooralsnog) de resources Catalogussen, Zaaktypen, Informatieobjecttypen, Roltypen, Statustypen en Zaakobjecten.
+
+Op dit moment bieden de resources alleen de mogelijkheid tot raadplegen van de gegevens (GET).
+
+### Resource: catalogus
```

Comment:
De naam van de resource is toch in meervoud? Geldt ook voor de hierna volgende resources.

Reply:
Alle resources zijn inmiddels meervoud